lib/pyfrc/mains/cli_deploy.py

```diff
@@ -230,6 +229,7 @@ def run(self, options, robot_class, **static_options):
             if not options.in_place:
                 # Restart the robot code and we're done!
                 sshcmd = "%(bash_cmd)s '" + \
+                         '%(replace_cmd)s;' + \
```

Comment:
is interpolation happening for replace_cmd?

Reply:
Whoops.